tests/ppx/react/expected/commentAtTop.res.txt

```diff
@@ -1,10 +1,14 @@
-@obj external makeProps: (~msg: 'msg, ~key: string=?, unit) => {"msg": 'msg} = "" // test React JSX file
+type props<'msg> = {@optional key: string, @optional ref: ReactDOM.Ref.currentDomRef, msg: 'msg} // test React JSX file
```

Comment:
I guess it's known and in the plans, but `ref` should not appear on normal examples.
Only on those using `forwardRef`.

Reply:
Good catch, thanks. I think it is not allowed in v3 either, even though passing the ref is working for react class component. I think it is reliable not to pass the ref unless it is for sure that it react class component. https://reactjs.org/docs/refs-and-the-dom.html#refs-and-function-components